src/Library/demos/Memory Monitor/main.js

```diff
@@ -0,0 +1,27 @@
+import Gio from "gi://Gio";
+
+const memory_monitor = Gio.MemoryMonitor.dup_default();
+
+const cache = new Map();
+cache.set("a", 1);
+cache.set("b", 2);
+cache.set("c", 3);
+
+memory_monitor.connect("low-memory-warning", (monitor, level) => {
+  // Use inequalities for checking as new levels may be added in the future
+  if (level >= Gio.MemoryMonitorWarningLevel.LOW) {
+    // Processes should free up unneeded resources
+    console.log("Warning Level: Low");
+    drop_caches();
+  } else if (level >= Gio.MemoryMonitorWarningLevel.MEDIUM) {
```

Comment:
This example is great overall! One comment: this should perhaps be a separate `if` rather than an `else if`, otherwise `drop_caches()` will only be called on levels between `LOW` and `MEDIUM`, rather than on all levels `>= LOW`.

Reply:
Actually wait, now that I think about it, won't it behave in a completely oppositive manner? For all levels ``>= LOW``, the first ``if`` condition would always be satisfied and it would never get to the other ``else if``(s) even if ``level >= MEDIUM``. I can't believe I missed this but still changing the ``else if`` to ``if`` should still fix the issue regardless.